.github/workflows/ci-test-go.yml

```diff
@@ -101,6 +101,15 @@ jobs:
         run: |
             go install gotest.tools/gotestsum@latest
             make test-unit
+            
+      - name: Analyze with SonarCloud
+        if: ${{ always() && inputs.run-tests && inputs.project-directory == '.' }}
```

Comment:
Once this is working for the parent go module we can do the same for each submodule.